config/ant-phase-verify.xml

```diff
@@ -170,5 +172,63 @@
     <fail if="checkstyle.failure.property"
           message="Checkstyle failed: ${checkstyle.failure.property}"
           />
+
+    <tstamp>
+      <format property="STARTED" pattern="dd/MM/yyyy hh:mm:ss aa"/>
+    </tstamp>
+    <echo level="info">Checkstyle started (checkstyle_input_checks.xml): ${STARTED}</echo>
+
+    <checkstyle config="${check.config_input}"
+                failOnViolation="true"
+                failureProperty="checkstyle.failure.property"
+                executeIgnoredModules="true"
+    >
+      <path>
+        <fileset dir="src"
+                 includes="test/resources/**/*.java,test/resources-noncompilable/**/*.java">
+          <!-- bad syntax, permanent exclude -->
+          <exclude name="**/InputAstTreeStringPrinter.java" />
+          <exclude name="**/InputSuppressionsStringPrinter.java"/>
+          <exclude name="**/InputMainIncorrectClass.java"/>
+          <exclude name="**/InputJavadocPropertiesGeneratorParseError.java"/>
+          <exclude name="**/InputMainFrameModelIncorrectClass.java"/>
+          <exclude name="**/InputBeforeExecutionExclusionFileFilterIncorrectClass.java"/>
+          <exclude name="**/InputJavaParser.java"/>
+          <!-- This check does not apply to grammar input files -->
+          <exclude name="**/grammar/**/*"/>
+          <!-- No need to check xpathmapper files -->
+          <exclude name="**/xpath/xpathmapper/**/*"/>
+          <!-- No need to check xpathquerygenerator files -->
+          <exclude name="**/xpath/xpathquerygenerator/**/*"/>
+          <!-- No need to check xpathfilegenerator files -->
+          <exclude name="**/xpathfilegenerator*/**/*"/>
+          <!-- No need to check main files -->
+          <exclude name="**/main/**/*"/>
+          <!-- No need to check javaparser files -->
+          <exclude name="**/javaparser/**/*"/>
+          <!-- No need to check treewalker files -->
+          <exclude name="**/treewalker/**/*"/>
+          <!-- No need to check gui files -->
+          <exclude name="**/gui/**/*"/>
+          <!-- No need to check api files -->
+          <exclude name="**/api/**/*"/>
+          <!-- No need to check ant files -->
+          <exclude name="**/ant/**/*"/>
```

Comment:
I've added these excludes, since it doesn't make sense to check for configs in these inputs, and could cause false positives in the future. Are there anymore directories that you can think of that we should exclude?

Reply:
We can find them later on